object/ldap_conn.go

```diff
@@ -191,6 +191,11 @@ func SyncSingleUser(
 	userExists := false
 	userExists = existingUserIDs[ldapUser.GetLdapUuid()]
 
+	localUserID, err := ldap_sync.GetLocalIDForExistingLdapUser(ldapUser.GetLdapUuid())
```

Comment:
Check GetLocalIDForExistingLdapUser() code. It returns error if no user is found. Which means you cant use it here.

Reply:
done